### ISSUE TITLE
FXIOS-1537 ⁃ Better detection of the mobile reporter

### ISSUE THREAD

**Issue by karlcow** [mozilla-mobile/firefox-ios#8063]
This is related to the [Report Site Issue Menu in Firefox iOS](https://github.com/mozilla-mobile/firefox-ios/issues/6498)

**Vision**

As a webcompat team member, I want to know when a reported site issue is coming from iOS specifically

**Acceptance Criteria**

* Report site issue option uses the Webcompat team's [better detection for the mobile reporter](https://github.com/webcompat/webcompat.com/pull/3548) which will help identify which browser the report came from

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1537)

**Comment by karlcow** [comment]
```diff
diff --git a/Shared/SupportUtils.swift b/Shared/SupportUtils.swift
index 32bd454e5..2089dac40 100644
--- a/Shared/SupportUtils.swift
+++ b/Shared/SupportUtils.swift
@@ -24,6 +24,6 @@ public struct SupportUtils {
         else {
             return nil
         }
-        return URL(string: "https://webcompat.com/issues/new?src=mobile-reporter&url=\(escapedUrl)")
+        return URL(string: "https://webcompat.com/issues/new?src=mobile-reporter&label=browser-firefox-ios&url=\(escapedUrl)")
     }
 }
```

**Comment by karlcow** [comment]
@dnarcese Is there a way for the webcompat team to speed this up. 
Currently webcompat.com is receiving a lot of reports from Firefox iOS 

<img width="417" alt="Capture d’écran 2021-05-03 à 10 49 00" src="https://user-images.githubusercontent.com/505230/116835453-8caaa000-abfd-11eb-8e7e-3dd6623f3458.png">


and we would love to have a better labeling. If necessary I can create a pull request for this issue. 

Are there other requirements in addition to modify this file (such as test files?)
https://github.com/mozilla-mobile/firefox-ios/blob/4f7debb9a02f86c2c35e984be31fe2345536df79/Shared/SupportUtils.swift#L27

Maybe here for the tests
https://github.com/mozilla-mobile/firefox-ios/blob/041c85483b1aa4a4e357d8bd9419550e4f20347a/SharedTests/SupportUtilsTests.swift



Our pull request has been deployed on our side. So we are ready.
https://github.com/webcompat/webcompat.com/pull/3548

**Comment by dnarcese** [comment]
@karlcow updating SupportUtils is the only change needed. 

I can take care of updating tests if it's needed.

**Comment by karlcow** [comment]
@dnarcese Thanks https://github.com/mozilla-mobile/firefox-ios/pull/8418

**Comment by SimonBasca** [comment]
Verified fix on main 493d272cca.
